pkg/printers/internalversion/printers.go

```diff
@@ -1328,11 +1330,14 @@ func printPersistentVolume(obj *api.PersistentVolume, options printers.PrintOpti
 	if obj.ObjectMeta.DeletionTimestamp != nil {
 		phase = "Terminating"
 	}
+	volumeMode := "<unset>"
+	if obj.Spec.VolumeMode != nil {
+		volumeMode = string(*obj.Spec.VolumeMode)
+	}
 
-	row.Cells = append(row.Cells, obj.Name, aSize, modesStr, reclaimPolicyStr,
-		string(phase), claimRefUID, helper.GetPersistentVolumeClass(obj),
-		obj.Status.Reason,
-		translateTimestampSince(obj.CreationTimestamp))
+	row.Cells = append(row.Cells, obj.Name, aSize, reclaimPolicyStr,
+		string(phase), claimRefUID, helper.GetPersistentVolumeClass(obj), volumeMode,
+		obj.Status.Reason, translateTimestampSince(obj.CreationTimestamp), modesStr)
```

Comment:
it looks like this moved access modes to the end, not volumemode

Reply:
Sorry, my mistake.  I found we lack pv unit tests, so I  have not found it.
I have sent an PR to add some Unit tests.

Reply:
Done!